lib/couchbaseX.js

```diff
@@ -306,6 +306,9 @@ class CouchbaseAccessor extends Accessor {
    * Result is a promise with `[id, rev]` or an error.
    */
   postWithId(id, data, options) {
+    if ('_cas' in data) {
+      delete data._cas;
```

Comment:
can you keep the original data unchanged? the caller might be surprised that `_cas` gets removed after the calling.

Reply:
modified